core/src/main/resources/config.yml

```diff
@@ -164,6 +164,16 @@ custom-skull-render-distance: 32
 # This option requires a restart of Geyser in order to change its setting.
 add-non-bedrock-items: true
 
+# Whether to allow custom blocks to be added.
+# This should only need to be disabled if using a proxy that does not use the "transfer packet" style of server switching.
+add-custom-blocks: true
```

Comment:
Not sure if we should even bother with this. Let's just combine it with the above setting? We can bump the config version if we have to.

Reply:
https://github.com/GeyserMC/Geyser/commit/b2d9095815450620c7efb021d8abaa283c049f46#diff-adb506ebeab4b5bdb97f04dd710496d2b52f64bf0ac4733ae694188996d68bd6

Reply:
https://github.com/GeyserMC/Geyser/commit/b2d9095815450620c7efb021d8abaa283c049f46#diff-34cf22e3d47cd52b85e8e9749b2bf87dd63929a459943083c16de23ddf8b8366
https://github.com/GeyserMC/Geyser/commit/b2d9095815450620c7efb021d8abaa283c049f46#diff-5482ed5484e9ade47524ad19eeea5751d2de4d5940e1f8830447476ff6000be2R40
https://github.com/GeyserMC/Geyser/commit/b2d9095815450620c7efb021d8abaa283c049f46#diff-053c2e7ce0e5bf9c89536553bc0119c622fbcc0c73c19391f784f21e0b4aa295
https://github.com/GeyserMC/Geyser/commit/b2d9095815450620c7efb021d8abaa283c049f46#diff-6f4b00915a86670b1c5b0550014514f6068795f5944136ecb8546fd7402d1e4a